docs/snap-clone.md

```diff
@@ -4,12 +4,12 @@
 - [Create CephFS Snapshot and Clone Volume](#create-cephfs-snapshot-and-clone-volume)
   - [Create CephFS SnapshotClass](#create-cephfs-snapshotclass)
   - [Create CephFS Snapshot](#create-cephfs-snapshot)
-  - [Restore CephFS Snapshot to a new PVC](#restore-cephfs-snapshot-to-a-new-pvc)
+  - [Restore CephFS Snapshot to a new PVC](#restore-cephfs-snapshot)
   - [Clone CephFS PVC](#clone-cephfs-pvc)
```

Comment:
@humblec can you also fix this link `Clone CephFS PVC`, I just realized its broken ?

Reply:
@pkalever sharp.. addressed in the latest patchset

Reply:
Thanks @humblec